applications/StructuralMechanicsApplication/custom_strategies/custom_builder_and_solver/explicit_builder_and_solver.hpp

```diff
@@ -0,0 +1,461 @@
+//
+//  Main authors:    Klaus B Sautter (based on the work of JMCarbonell)
+//					 
+
+#if !defined(KRATOS_EXPLICIT_BUILDER_AND_SOLVER )
+#define  KRATOS_EXPLICIT_BUILDER_AND_SOLVER
+
+
+/* System includes */
+#include <set>
+
+#ifdef _OPENMP
+#include <omp.h>
+#endif
+
+/* External includes */
+
+
+/* Project includes */
+#include "includes/define.h"
+#include "solving_strategies/builder_and_solvers/builder_and_solver.h"
+#include "includes/model_part.h"
+#include "includes/element.h"
+
+namespace Kratos
+{
+
+/**@name Kratos Globals */
+/*@{ */
+/*@} */
+/**@name Type Definitions */
+/*@{ */
+/*@} */
+/**@name  Enum's */
+/*@{ */
+/*@} */
+/**@name  Functions */
+/*@{ */
+/*@} */
+/**@name Kratos Classes */
+/*@{ */
+
+template<class TSparseSpace,
+         class TDenseSpace, //= DenseSpace<double>,
+         class TLinearSolver //= LinearSolver<TSparseSpace,TDenseSpace>
+         >
+class ExplicitBuilderAndSolver
+    : public BuilderAndSolver< TSparseSpace, TDenseSpace, TLinearSolver >
+{
+public:
+    /**@name Type Definitions */
+    /*@{ */
+    KRATOS_CLASS_POINTER_DEFINITION( ExplicitBuilderAndSolver );
+
+    typedef BuilderAndSolver<TSparseSpace, TDenseSpace, TLinearSolver> BaseType;
+
+    typedef typename BaseType::TSchemeType TSchemeType;
+
+    typedef typename BaseType::TDataType TDataType;
+
+    typedef typename BaseType::DofsArrayType DofsArrayType;
+
+    typedef typename BaseType::TSystemMatrixType TSystemMatrixType;
+
+    typedef typename BaseType::TSystemVectorType TSystemVectorType;
+
+    typedef typename BaseType::LocalSystemVectorType LocalSystemVectorType;
+
+    typedef typename BaseType::LocalSystemMatrixType LocalSystemMatrixType;
+
+    typedef typename BaseType::TSystemMatrixPointerType TSystemMatrixPointerType;
+
+    typedef typename BaseType::TSystemVectorPointerType TSystemVectorPointerType;
+
+    typedef typename BaseType::NodesArrayType NodesArrayType;
+
+    typedef typename BaseType::ElementsArrayType ElementsArrayType;
+
+    typedef typename BaseType::ConditionsArrayType ConditionsArrayType;
+
+    typedef typename BaseType::ElementsContainerType ElementsContainerType;
+
+    /*@} */
+    /**@name Life Cycle
+     */
+    /*@{ */
+
+    /** Constructor.
+     */
+    ExplicitBuilderAndSolver(
+        typename TLinearSolver::Pointer pNewLinearSystemSolver)
+        : BuilderAndSolver< TSparseSpace, TDenseSpace, TLinearSolver >(pNewLinearSystemSolver)
+    {
+    }
+
+    /** Destructor.
+     */
+    virtual ~ExplicitBuilderAndSolver()
+    {
+    }
+
+
+    /*@} */
+    /**@name Operators
+     */
+    /*@{ */
+
+
+    //**************************************************************************
+    // this does not build a matrix but is used to create the LHS
+    // in explicit time integration each node is updated and thus
+    // does not need to assemble any matrix
+    //**************************************************************************
```

Comment:
this is not doxygen compatible
I think @loumalouomega has a good doxygen decumentation in the ContactStructuralMechanicsApp

Reply:
I try, right now experimenting with Doxygen for launching nighly https://loumalouomega.github.io/html/